addons/binding/org.openhab.binding.lgwebos/src/main/java/org/openhab/binding/lgwebos/internal/LGWebOSHandlerFactory.java

```diff
@@ -45,10 +48,14 @@ protected void unbindDiscovery(LGWebOSDiscovery discovery) {
     }
 
     @Override
-    protected ThingHandler createHandler(Thing thing) {
+    protected @Nullable ThingHandler createHandler(Thing thing) {
+        LGWebOSDiscovery lgWebOSDiscovery = discovery;
+        if (lgWebOSDiscovery == null) {
```

Comment:
@kaikreuzer is this a correct dependency?

Reply:
Not really - we usually have the dependency just the other way round, i.e. the discovery can depend on a handler.

But if I remember correctly, I already discussed this with @sprehn once and we agreed on this "non-standard" dependency here - after all, the dependency was already here before and hasn't been introduced by this PR.

Reply:
Great, it was indeed not as new as a looked at a first glance :-)